### PR TITLE
Make use of django_storage

### PR DESCRIPTION
**Pull request by vkosuri**
Currently the django_storage adapter tightly coupled with django_chatterbot. By introduce this fix, others can reuse storage adapter.

@gunthercox please provide your comments/suggestions on this PR.

### CODE REVIEW

File: chatterbot/storage/django_storage.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import json
 from chatterbot.storage import StorageAdapter
 
-
```
Comment (gunthercox) [inline chatterbot/storage/django_storage.py]: This line should not have been deleted. Python's style guide specifies that there should be two blank lines between imports and the a class definition. Once this is fixed I will merge this pull request.

Comment (vkosuri) [inline chatterbot/storage/django_storage.py]: Thanks Master!. I'll made my changes through [PyCharm](https://www.jetbrains.com/pycharm/). I found other issues with Train  ``python manage.py train`` with ``import_module`` option isn't working train command. it was failing always. The workaround I have made like this

I tried figured out bug, i couldn't, could you please let me know why it failin and how to solve this problem?

``` Python
CHATTERBOT_TRAIN = {
    'name': 'ADTRAN Chatter Bot',
    'trainer': 'chatterbot.trainers.ChatterBotCorpusTrainer',
    'training_data': [
        './data/greetings.corpus.json'
    ],
    'storage_adapter': 'chatterbot.storage.DjangoStorageAdapter',
    'django_app_name': 'howdy',
    'use_django_models': True
}
# ChatterBot settings
CHATTERBOT = {
    'name': 'Test Chatter Bot',
    'logic_adapters': [
        {
            'import_path': 'chatterbot.logic.BestMatch',
        },
        {
            'import_path': 'chatterbot.logic.LowConfidenceAdapter',
            'threshold': 0.75,
            'default_response': 'I am sorry, but I do not understand.'
        },
        {
            'import_path': 'chatterbot.logic.SpecificResponseAdapter',
            'input_text': 'Help me!',
            'output_text': 'Ok, here is a link: http://chatterbot.readthedocs.io/en/stable/'
        }
    ],
    'filters': [
            'chatterbot.filters.RepetitiveResponseFilter'
    ],
    'storage_adapter': 'chatterbot.storage.DjangoStorageAdapter',
    'django_app_name': 'howdy',
    'use_django_models': True
}
```

Comment (vkosuri) [inline chatterbot/storage/django_storage.py]: And also the response and statements are still reversed in django_storage adapter

Comment (gunthercox) [inline chatterbot/storage/django_storage.py]: I'm not sure about the `import_module`, feel free to open a new issue for it.

**Comment by vkosuri** [comment]
@gunthercox Master, any plans to release chatterbot minor version this week?

**Comment by gunthercox** [comment]
Yes, possibly this weekend.

**Comment by vkosuri** [comment]
@gunthercox Any updates?

**Comment by gunthercox** [comment]
@vkosuri Apologies. I haven't had a chance to get around to drafting a new release yet. There is still a few possible issues that I want to investigate with the new SQL Alchemy adapter before doing the release.

**Pull request merged**